.buildkite/test-pipeline.yaml

```diff
@@ -117,6 +117,17 @@ steps:
   - pytest -v -s entrypoints/test_chat_utils.py
   - pytest -v -s entrypoints/offline_mode # Needs to avoid interference with other tests
 
+# platform plugin test need a single pipeline, since it needs to install a new fake platform plugin
+- label: Platform Plugin Test # < 1min
+  working_dir: "/vllm-workspace/tests"
+  fast_check: true
+  mirror_hardwares: [amd]
+  source_file_dependencies:
+  - vllm/
+  commands:
+  - pip install -e ./plugins/vllm_add_dummy_platform
+  - pytest -v -s platform/test_platform_plugin.py
+
```

Comment:
Perhaps it would be better to have a dedicated Plugins tests that includes the model plugins as well? It would be cost-inefficient to spin up an instance just for this one test.

Reply:
I notice that the model plugin is tested in `Entrypoints Test ` already. https://github.com/vllm-project/vllm/blob/a491d6f535d96939d17e5290991dc975495c9580/.buildkite/test-pipeline.yaml#L109

Of cause it can be moved here.

And in the future, once the platform plugin works well, we can install a real plugin for test and more test will be added, in that case, the new pipeline may be worth to have. How about to keep it now?

Reply:
Let's move those tests under Plugin Tests as well.

Reply:
Got it